st2common/tests/unit/test_util_actionalias_matching.py

```diff
@@ -0,0 +1,129 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+import mongoengine as me
+
+from st2common.constants.types import ResourceType
+from st2common.models.db import stormbase
+import st2common.util.actionalias_matching as matching
+
+
+class MemoryActionAliasDB(stormbase.StormFoundationDB, stormbase.ContentPackResourceMixin,
```

Comment:
Just curious - why do we need this class here?

Can't we re-use one from the existing model? If we have another model here, it means another place to maintain in sync and things will get out of date...